iron-fish/ironfish
# Rahul/ifl 1498 show enabled state in wallet status

## Summary
Adding the wallet enabled state to the status command: 


https://github.com/iron-fish/ironfish/assets/13268167/6c76bfe9-8a81-421f-a13c-9851347e4d3d



## Testing Plan

1. Set the config enableWallet flag to false/ true 
2. Run `ironfish status` and you should see the state reflected 

## Documentation

Does this change require any updates to the Iron Fish Docs (ex. [the RPC API
Reference](https://ironfish.network/docs/onboarding/rpc/chain))? If yes, link a
related documentation pull request for the website.

```
[ ] Yes
```

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```
